viz_scripts/generic_timeseries.ipynb

```diff
@@ -223,10 +237,7 @@
     "    alt_text = store_alt_text_timeseries(plot_data, file_name, plot_title)\n",
     "\n",
     "try:\n",
-    "    if (len(dynamic_labels)) > 0:\n",
-    "        plot_emission_per_week(emissions, 'kg', active_users)\n",
-    "    else:\n",
-    "        plot_emission_per_week(emissions, 'lb', active_users)\n",
+    "    plot_emission_per_week(emissions, weight_unit, active_users)\n",
     "except:\n",
```

Comment:
Nice! that we were able to get rid of these repetitive `if/then` checks

Reply:
Your centralization suggestions have been extremely useful to centralization and reducing repeated code!